### PR TITLE
Fixes sprint creation

### PR DESCRIPTION
**Pull request by brunapinos**
## Proposed Changes
Changes function `getReleases` and `getSprint` to make them async. Also, changes the variable `releaseId`, to allow sprint creation in the right release.

Related to sprints being displayed even though they don't belong to the release selected, it has been fixed adding the fucntion `getSprints` after setting the right releaseId.

Solves #362, #363 and #325 

## Type of Changes

What type of change this Pull Request brings to Falko?

- [x] Bugfix
- [ ] New Feature

## Checklist

- [x] This Pull Request has a significant name.
- [ ] The build is okay (tests, code climate).
- [x] This Pull Request mentions a related issue.

### CODE REVIEW

**Review by alaxalves — APPROVED** [review]

**Review by alaxalves — DISMISSED** [review]

File: src/components/Releases/Releases.vue [hunk 1]
```diff
@@ -130,35 +131,46 @@ export default {
...
+        return;
+      }
+
+      try {
+        let response = await HTTP.get(`releases/${this.releaseId}/sprints`, { headers });
+        if(response.data.length !== 0) {
+          this.sprints = response.data;
+          return;
+        }
+        this.sprints = [];
```
Comment (MatheusRich) [inline src/components/Releases/Releases.vue]: Should this have a return after this line too? Why only inside the `if` condition?

File: src/components/Releases/Releases.vue [hunk 2]
```diff
@@ -130,35 +131,46 @@ export default {
...
-        });
+      if(this.isReleasesEmpty()) {
+        return;
+      }
+
+      try {
+        let response = await HTTP.get(`releases/${this.releaseId}/sprints`, { headers });
+        if(response.data.length !== 0) {
+          this.sprints = response.data;
+          return;
```
Comment (alaxalves) [inline src/components/Releases/Releases.vue]: @brunapinos Is this really supposed to be here? :thinking:

File: src/components/Releases/Releases.vue (hunk 1 above)
Comment (brunapinos) [reply src/components/Releases/Releases.vue]: This if condition guarantees the returning of sprints not empty. If there is no sprint, I set explicitly that the `this.sprints` is an empty list, without it, the code catches the previous sprint wrongly.

File: src/components/Releases/Releases.vue [hunk 3]
```diff
@@ -79,6 +79,7 @@ import ReleaseCard from './ReleaseCard.vue';
 import SprintCard from '../Sprints/SprintCard.vue';
 import AddSprint from '../Sprints/AddSprint.vue';
 import dateConvert from '../../mixins/dateConvert';
+import { release } from 'os';
```
Comment (matheussbernardo) [inline src/components/Releases/Releases.vue]: Where this import come from?

Comment (brunapinos) [reply src/components/Releases/Releases.vue]: This import is not necessary, I will remove it

**Review by alaxalves — APPROVED** [review]

**Pull request merged**